### PR TITLE
Print error to console and don't spam notify.

### PR DESCRIPTION
**Pull request by thegrb93**
So you can actually see the error.

### CODE REVIEW

**Comment by Xandaros** [comment]
Looks good to me; fix the code style and I'll merge it in.

**Comment by awilliamson** [comment]
Put the newline variable inline if it's not used anywhere else. Then I'm happy.

**Comment by Xandaros** [comment]
It is used twice, inlining it wouldn't look too good...

**Comment by thegrb93** [comment]
Okay I fixed it and squashed it.

File: lua/starfall/sflib.lua [hunk 1]
```diff
@@ -644,6 +644,13 @@ else
             return
         end
 
+		print( msg )
+		
+		local newline = string.find( msg, "\n" )
+		if newline then
+			msg = string.sub( msg, 1, newline-1 )
```
Comment (Xandaros) [inline lua/starfall/sflib.lua]: Can you put spaces around the minus?
Sorry

Otherwise, it looks good

**Comment by Xandaros** [comment]
By the way, instead of committing and squashing, you can use `git commit --amend` to edit the latest commit. (It will prompt for the commit message again, you can just leave it as-is)

It will require a force push afterwards.

**Comment by thegrb93** [comment]
Thanks for the tip.

**Comment by thegrb93** [comment]
It screwed up my tabs for some reason though.

**Comment by thegrb93** [comment]
Never mind, it was because some of the file was tabs and some was spaces. I replaced all the spaces with tabs.

**Comment by thegrb93** [comment]
It's making the difference comparison act weird but I only changed the spaces into tabs.

**Comment by thegrb93** [comment]
I'm closing this branch.

**Pull request closed without merging**